videostore/videostore.go

```diff
@@ -396,6 +401,77 @@ func (vs *videostore) Fetch(_ context.Context, r *FetchRequest) (*FetchResponse,
 	return &FetchResponse{Video: videoBytes}, nil
 }
 
+func (vs *videostore) FetchStream(ctx context.Context, r *FetchRequest, emit func(video.Chunk) error) error {
```

Comment:
I know it's a lot of work, but it could be useful to use one of our artifacts and make sure that streaming it via emit works